src/python/gem5/components/boards/arm_board.py

```diff
@@ -80,6 +82,7 @@ class ArmBoard(ArmSystem, AbstractBoard, KernelDiskWorkload):
 
     **Limitations**
     * stage2 walker ports are ignored.
+    * KVM cores only work with VExpress_GEM5_V1
```

Comment:
Maybe we should force the simulation to fail if the number of cores is bigger than 8 and the VExpress version is GEM5_V1.

Reply:
Sounds like a good idea. I'll update this on the next patch. We need to check the number of cores, VExpress class, and the bootloader file (has to be bootloader (V1) compatible with VExpress_GEM5_V1).

Reply:
I've updated this in the new patchset by adding a fatal when the user tries to simulate anything other than VExpress_GEM5_V1.